### DISCUSSION TITLE
Global EXP Modifier

### DISCUSSION THREAD

**Discussion by Bioblaze** [AscensionGameDev/Intersect-Engine#1503] [Ideas]
**Is your feature request related to a problem? Please describe.**
Global EXP Modifier for a Server

**Describe the solution you'd like**
Add Supported Functionality for a Global EXP Modifier
[https://www.ascensiongamedev.com/topic/5808-beta-6-global-exp-modifierw-console-support/](https://www.ascensiongamedev.com/topic/5808-beta-6-global-exp-modifierw-console-support/)

**Additional context**

**Comment by blinkuz** [comment]
It is a very good utility, and it can be used for many types of games.

**Comment by Bioblaze** [comment]
I made a PR about this Topic: https://github.com/AscensionGameDev/Intersect-Engine/pull/592

**Comment by Bioblaze** [comment]
this was a while ago, wonder if I should update it too the newest version.
